README.md

```diff
@@ -17,36 +17,36 @@ snowflake connector.
+|---------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|--------------------------------------------------------------------|
+| `connection`        | Snowflake connection string. **Important:** Schema is required.                                                                                                                                                                                 | true     | "user:password@my_organization-my_account/mydb/schema" or "username[:password]@hostname:port/dbname/schemaname" |
+| `table`             | The table the connector will read records from.                                                                                                                                                                                                 | true     | "users"                                                            |
+| `columns`           | Comma-separated list of column names that should be included in each record payload. Default is `false` which translates to `All columns`.                                                                                                      | false    | "id,name,age"                                                      |
+| `primaryKeys`       | Comma-separated list of column that records should use for their `Key` fields.                                                                                                                                                                  | false    | "id,name"                                                          |
+| `orderingColumn`    | The column name the connector will use for ordering rows. **Important:** Values must be unique and suitable for sorting. Otherwise, the snapshot will not work.                                                                                 | true     | "id"                                                               |
+| `snapshot`          | Whether or not the plugin will take a snapshot of the entire table before starting cdc mode. Default is `true`. Other possible value is `false`.                                                                                                | false    | "false"                                                            |
+| `batchSize`         | Size of batch. **Important:** Please don't update this variable after the pipeline starts, it will cause problems with tracking position. Default is `1000`.                                                                                    | false    | "1000"                                                             |
```

Comment:
Oddly, this does not seem reflective of what is show as the `default` here: https://github.com/conduitio-labs/conduit-connector-snowflake/blob/main/source/config_paramgen.go

Reply:
yeah that should be fixed (maybe Mario can take this one @raulb 👀, cuz commons needs to be used for the new version of paramgen, or I can take this on next week too!)
another weird thing is that all the params have the prefix `snowflake.`, also no idea why!

Reply:
@justmisosoup @maha-hajja Default is wrong because of this https://github.com/conduitio-labs/conduit-connector-snowflake/blob/f58b5db2ad6987894c1d198dfdc546bfc8890fee/source/config.go#L33 (which is where the automatically generated configuration and values are coming from).

The `snowflake` configuration prefix are also coming from there. Notice the `snowflake.url`, etc...

Reply:
@justmisosoup I took the liberty to fix this default value from the source and adding some housekeeping for the future. Hope you don't mind. https://github.com/conduitio-labs/conduit-connector-snowflake/pull/121/commits/20b14fd29815973b5de8de77a27e5635852cdeb5

I'll go ahead and ship this

Reply:
Thank you!